src/nedap/utils/spec/impl/check.cljc

```diff
@@ -44,5 +47,9 @@
                                    :faulty-value        x-quoted#
                                    :faulty-value-object x#
                                    :quoted-faulty-value x-quoted#
-                                   :explanation         (~explain spec# x#)})))))
+                                   :explanation         (~explain spec# x#)})]
+                    (when (and (not ~cljs)
+                               (pos-int? *print-stack-frames*))
+                       (clojure.stacktrace/print-stack-trace ex# *print-stack-frames*))
```

Comment:
It seems bit of a weird api to expect the client to set an arbitrary number. Are you simply using a high enough number that it's unlikely that stacktraces will be trimmed?

`io.aviso.exception` might be a nice alternative as it has the :omit and :terminate directives.

Reply:
Personally I've used 5, which is usually enough to provide the context needed. I chose clj-stacktrace because of the low impact dependency wise (e.g. no extra deps)

Reply:
Ok, doesn't seem a big deal particularly when this feature addresses a pretty specific use case (normally apps handle exceptions just fine).

When documenting this feature,  please consider emphasizing that.

Reply:
Added 👍